recipe/meta.yaml

```diff
@@ -1,27 +1,34 @@
 {% set name = "pmdarima" %}
-{% set version = "1.8.4" %}
+{% set version = "1.8.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 92b17c5baf2f0952f69f0cfae0511ff396bb281387ff315f162cb8f919859b0f
+  sha256: 090a7e195e849059e6b7f9908d80539f63c0771d8f9c5da44472960f17fcc9fa
 
 build:
-  number: 1
-  skip: true  # [py2k]
+  number: 0
+  # Unsatisfiable dependencies on win32 with python 3.10
+  skip: true  # [py<37 or (win32 and py==310)]
```

Comment:
Please change selector to `[py<37 or (win32 and py>=310)]`.

Reply:
Fixed